servicetalk-dns-discovery-netty/build.gradle

```diff
@@ -26,10 +26,11 @@ dependencies {
   implementation project(":servicetalk-transport-netty")
   implementation project(":servicetalk-transport-netty-internal")
   implementation "com.google.code.findbugs:jsr305:$jsr305Version"
-  implementation "io.netty:netty-resolver-dns:$nettyVersion"
+  implementation(platform("io.netty:netty-bom:$nettyVersion"))
+  implementation "io.netty:netty-resolver-dns"
   implementation "org.slf4j:slf4j-api:$slf4jVersion"
 
-  runtimeOnly "io.netty:netty-resolver-dns-native-macos:$nettyVersion:osx-x86_64"
+  runtimeOnly( group:"io.netty", name:"netty-resolver-dns-native-macos", classifier:"osx-x86_64")
```

Comment:
Switching to the "map" syntax is necessary for omitting version as empty version in `foo:bar::classifier` is not allowed.